app/services/search/search_builder.rb

```diff
@@ -53,10 +56,15 @@ def build_suggestions
     end
   end
 
-  def call_algolia_search
-    search = Search::AlgoliaSearchRequest.new(search_params)
+  def call_search
+    search = if any?
```

Comment:
What does `any?` do? Google didn't enlighten me.

Reply:
It's a pre-existing method in the class if you expand above/below the diff